source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/locale-input/-page.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  interactor,
+  Interactor,
+  hasClass,
+  scoped,
+  collection,
+  clickable,
+  isPresent,
+  triggerable,
+  fillable,
+  text,
+  value,
+  attribute,
+} from '@bigtest/interactor';
+import { when } from '@bigtest/convergence';
+import { Simulate } from 'react-dom/test-utils';
+import { assert } from 'chai';
+import { wait } from 'tests/helpers';
+
+class LocaleInput extends Interactor {
+  static defaultScope = '[data-test-locale-input-container]';
+
+  async searchAndSelect(searchTerm: string) {
+    await when(() => assert(this.isPresent, `\nexpected language picker to be visible\n`));
+
+    await this.input.fill(searchTerm);
+    // bigtest blur/focus don't work?
+    Simulate.blur(this.input.$root);
+    Simulate.focus(this.input.$root);
```

Comment:
The react-autosuggest input activates strangely in a programmatically driven environment, not sure what that's about -- whether it'd be a bug with them, or what, but I took this technique from their own tests.